org-code-javabuilder/protocol/src/main/java/org/code/protocol/DelegatingAssetFileHelper.java

```diff
@@ -0,0 +1,40 @@
+package org.code.protocol;
+
+import java.io.FileNotFoundException;
+
+/**
+ * An {@link AssetFileHelper} that delegates to either the standard AssetFileHelper implementations
+ * of its methods, or to a {@link ContentManager}. Used to temporarily support two code paths.
+ */
+public class DelegatingAssetFileHelper extends AssetFileHelper {
```

Comment:
do we have a work item to remove this and `DelegatingFileManager` once they are no longer needed?

Reply:
We have [this](https://codedotorg.atlassian.net/browse/JAVA-451) task to generally cleanup unused code after migrating - I'll mention it there.